sources/models/src/aws-k8s-1.21/mod.rs

```diff
@@ -4,9 +4,9 @@ use std::collections::HashMap;
 
 use crate::modeled_types::Identifier;
 use crate::{
-    AwsSettings, BootstrapContainer, HostContainer, KernelSettings, KubernetesSettings,
```

Comment:
I'm putting this here, since there isn't another place. Shouldn't you add this setting to NVIDIA variants as well?